packages/i18n-utils/src/utils.ts

```diff
@@ -226,3 +226,26 @@ export function isMagnificentLocale( locale: string ): boolean {
 export function isTranslatedIncompletely( locale: string ) {
 	return getLanguage( locale )?.isTranslatedIncompletely === true;
 }
+
+/**
+ * Removes the Locale from the given path and returns the real path.
+ *
+ * @param {string}  path  Original path
+ * @returns {string} path The real path (with locale removed)
+ */
+export function getRealPathName( path: string ) {
```

Comment:
Should we think of a name that better expresses the intentions here?

Reply:
@tyxla I introduced a new function `removeLocaleFromPathLocaleInFront` similar to `removeLocaleFromPath` and got rid of the `getRealPath` method: https://github.com/Automattic/wp-calypso/pull/69731#discussion_r1087670783